llpc/CMakeLists.txt

```diff
@@ -391,6 +393,13 @@ add_executable(amdllpc tool/amdllpc.cpp)
 add_dependencies(amdllpc llpc_standalone_compiler)
 target_link_libraries(amdllpc PRIVATE llpc_standalone_compiler)
 set_compiler_options(amdllpc ${LLPC_ENABLE_WERROR})
+#if USING_STATIC_SPVGEN
+if(USING_STATIC_SPVGEN)
+    target_link_libraries(amdllpc PRIVATE spvgen_static)
+    add_compile_definitions(amdllpc PRIVATE SH_EXPORTING)
+    add_compile_definitions(amdllpc PRIVATE USING_STATIC_SPVGEN)
```

Comment:
You can provide multiple definitions at once: https://cmake.org/cmake/help/latest/command/target_compile_definitions.html